src/ipyaladin/utils/_coordinate_parser.py

```diff
@@ -35,6 +44,45 @@ def parse_coordinate_string(string: str) -> SkyCoord:
     return SkyCoord(ra=ra, dec=dec, frame="icrs")
 
 
+def _from_name_on_planet(string: str, body: str) -> SkyCoord:
+    """Get coordinates from a name on a planetary body.
+
+    Parameters
+    ----------
+    string : str
+        The name of the feature.
+    body : str
+        The planetary body to use for the coordinates.
+
+    Returns
+    -------
+    SkyCoord
+        An `astropy.coordinates.SkyCoord` object representing the coordinates.
+    """
+    url = (
+        f"https://alasky.cds.unistra.fr/planetary-features/resolve"
+        f"?identifier={string.replace(' ', '+')}&body={body}&threshold=0.7&format=json"
+    )
+    request = requests.get(url)
+    if request.status_code != requests.codes.ok:
+        raise ValueError(f"Invalid coordinate string: {string}")
+    data = request.json()
+    identifier = data["data"][0][1]
+    lat = data["data"][0][5]
+    lon = data["data"][0][6]
+    system = data["data"][0][11]
+    if identifier != string:
+        warnings.warn(
+            f"Nothing found for '{string}', but found close"
+            f" name '{identifier}'. Moving to {identifier}.",
+            NameResolverWarning,
+            stacklevel=2,
+        )
+    if "+West" in system:
+        lon = 360 - lon
+    return SkyCoord(ra=lon, dec=lat, frame="icrs", unit="deg")
```

Comment:
maybe add a comment to signal that for the moment the frame 'icrs' will be changed or return just a tuple (lon, lat) instead of a skycoord